README.md

```diff
@@ -1 +1,52 @@
-# Hello-Micronaunts
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+
```

Comment:
Here you can add instructions on how to set up the project. For instance, you can provide the link for cloning your project, the text editor and browser to be used and how to open or start the project. You can use the image below as a guidance

![Screenshot from 2022-09-20 22-06-16](https://user-images.githubusercontent.com/65233860/191373514-7be98db9-edd0-443f-a7a9-2c476a9c1ba5.png)